statocles-site/blog/2024/12/16/pitch-detection/index.markdown

```diff
@@ -0,0 +1,243 @@
+---
+title: Day 16: Pitch detection
+disable_content_template: 1
+tags:
+    - signal processing
+    - phonetics
+    - speech
+author: José Joaquín Atria
+images:
+  banner:
+    src: 'banner.jpg'
+    alt: A close up of a person's ear
+    data:
+      attribution: |-
+        <a href="https://www.pexels.com/photo/close-up-photo-of-a-person-s-ear-8092973/">Close-Up Photo of a Person's Ear</a> Image Credit: Kaboompics.com, <a href="https://www.pexels.com/license/">free to use</a>
+data:
+  bio: jjatria
+  description: |
+    An explanation of (part of) Praat's pitch detection algorithm using PDL
+---
+
+In a previous life I spent most of my time (free and otherwise) thinking about
+phonetics and the analysis of speech sounds. Like most scientific research, it
+was slow and tedious and oddly rewarding.
+
+Back then I was already in a steady drift towards spending more time playing
+with computers than with my research, so a lot of my efforts went into
+studying [Praat](https://www.fon.hum.uva.nl/praat), a piece of software you've
+almost certainly never heard of, but that has probably been used in most of
+the phonetics research you've also never heard of.
+
```

Comment:
Could you insert a section break `---` here, so the RSS feed summary (and `blog/` page) are nice and short? Creates interest that will force any reasonable person to read the rest :-)

Reply:
Done!